### ISSUE TITLE
avoid columns being sorted ??

### ISSUE THREAD

**Issue by pulketo** [wlezzar/jtab#4]
I guess that something is sorting columns alphabetically, but i don't want that, is there anyway to avoid this?
the input is:
[
    {
        "hostname": "xxxxx",
        "id": 191,
        "parent": yyy,
        "user": "zzz",
        "pass": "fff",
        "utype": "admin",
        "extra": "ggg",
        "stamp": "2022-03-03 19:10:17"
    }
]
output:
```
extra                    │ hostname │ id  │ parent │ pass                             │ stamp               │ user       │ utype
```
since now, thank you for your time.

**Comment by Lipen** [comment]
@pulketo this is due to `serde` not preserving the order of keys by default: [see docs](https://docs.rs/serde_json/latest/serde_json/enum.Value.html#variant.Object). You can "workaround" this by manually specifying the `preserve_order` feature for `serde_json` during the installation of `jtab`:
```sh
cargo install --git https://github.com/wlezzar/jtab --features serde_json/preserve_order
```

*Alternatively*, clone `jtab` repo and modify `Cargo.toml`:
```toml
[dependencies]
serde_json = { version = "1.0", features = ["preserve_order"] }
```
and install it:
```sh
cargo install --path .
```

**Comment by wlezzar** [comment]
Thanks for raising the issue @pulketo and thanks @lipen for suggesting the fix!

I applied your suggestion and this should be available in the version 0.5.1.
